build.gradle

```diff
@@ -14,13 +14,13 @@ buildscript {
         android_components: '15.0.0',
         android_gradle_plugin: '3.5.3',
         android_maven_publish_plugin: '3.6.2',
-        coroutines: '1.3.0',
+        coroutines: '1.3.7',
         dokka: '0.9.17',
         jna: '5.2.0',
         junit: '4.12',
         mockito: '2.24.5',
         mockwebserver: '3.10.0',
-        kotlin: '1.3.50',
+        kotlin: '1.3.72',
```

Comment:
In general, I don't think we should target Fenix requirements for our versions. These requirements, for example, are different from the one A-C uses and often bleeding edge.

A safer path would be to target what A-C does (e.g. [here](https://github.com/mozilla-mobile/android-components/blob/21816c76c7b6c6fa697338b61198fe0b509afcd0/buildSrc/src/main/java/Dependencies.kt#L7)).

Reply:
Not a problem, as I mentioned in the Bugzilla comment, I've updated this PR to match A-C rather than Fenix.